Files/Views/LayoutModes/DetailsLayoutBrowser.xaml

```diff
@@ -596,13 +596,12 @@
                                     x:Name="ItemStatusGrid"
                                     Grid.Column="4"
                                     Width="{Binding ColumnsViewModel.StatusColumn.Length.Value, ElementName=PageRoot, Mode=OneWay}"
-                                    Padding="12,0,0,0"
                                     HorizontalAlignment="Stretch"
                                     VerticalAlignment="Center"
                                     Visibility="{Binding ColumnsViewModel.StatusColumn.Visibility, ElementName=PageRoot, Mode=OneWay}">
                                     <FontIcon
                                         x:Name="CloudDriveSyncStatusGlyph"
-                                        HorizontalAlignment="Left"
+                                        HorizontalAlignment="Center"
```

Comment:
The Status column header should be centered as well
<img src="https://user-images.githubusercontent.com/9673091/127751065-1fb68353-4acb-4330-bd74-8c8befc4f2e4.png" width="600"/>

Reply:
![image](https://user-images.githubusercontent.com/59544401/127752034-9caec3ce-ceaf-45d6-b3c6-4f9aec768594.png)

Reply:
I think we should keep it left aligned to match the other columns.

Reply:
> I think we should keep it left aligned to match the other columns.

Just the header or both?

Reply:
I think both should be left aligned, what we can do to improve the design is make the icons a little smaller and closer in line with the icons in File Explorer.